code/modules/mob/living/carbon/xenomorph/egg_item.dm

```diff
@@ -106,6 +106,12 @@
 		to_chat(user, SPAN_XENOWARNING("[src] cannot be planted inside a vehicle."))
 		return
 
+	for(var/obj/object in T.contents)
+		var/obj/effect/alien/egg/xeno_egg = /obj/effect/alien/egg
+		if(object.layer > initial(xeno_egg.layer))
+			to_chat(user, SPAN_XENOWARNING("[src] cannot be planted below objects that would obscure it."))
```

Comment:
Ehh i don't know, what if the object gets put after? I wasn't a fan of previous one but i think it made more sense

Reply:
Missed this earlier; I don't think it'll be a problem, there aren't many things that'd layer over an egg's sprite that xenoes can create or move.